sway-lib-std/src/u256.sw

```diff
@@ -88,4 +107,268 @@ impl U256 {
     pub fn bits() -> u32 {
         256
     }
+
+    /// Get 4 64 bit words from a single U256 value.
+    fn decompose(self) -> (u64, u64, u64, u64) {
```

Comment:
Hmm. So ideally this would use from and into, but that requires additional compiler work IIRC. I wouldn't be against just keeping it as-is until then, since it's going away in the not too distant future.